tests/src/JIT/superpmi/superpmicollect.cs

```diff
@@ -199,17 +205,20 @@ private static void RunTest(string testName)
 
             if (Global.IsWindows)
             {
+                Console.WriteLine("================O=========="); 
```

Comment:
Did you meant to keep these debug statements?

Reply:
Nope. Will remove.